### ISSUE TITLE
Prova..

### ISSUE THREAD

**Issue by tikaoo** [cwi-reset/edicao-04-level-1#360]
Será possível utilizar duas telas durante a prova?

**Comment by Askalor** [comment]
Acho que a pergunta mais adequada seria se será possível usar materiais de consultas ou coisas do tipo, não? Já que essa seria uma das vantagens de se ter duas telas. Além de poder ter anotações em papel também.

Também possuo dois monitores e tenho essa duvida

**Comment by igorfaustino** [comment]
> Acho que a pergunta mais adequada seria se será possível usar materiais de consultas ou coisas do tipo, não? Já que essa seria uma das vantagens de se ter duas telas. Além de poder ter anotações em papel também.
> 
> Também possuo dois monitores e tenho essa duvida

Uma boa pergunta, pois faço muitas anotações em caderno e também no VS Code. Não sei se o intuito de estar com a web cam ligada seria para ver se somos nós mesmos ou se estamos utilizando meios de consulta como caderno e celular.

Mas acho que estamos muito ansiosos, vamos ter calma e relaxar.

**Comment by Tayling-Ng** [comment]
Boa Noite! Tem que ficar com a câmera ligada?

**Comment by JaquelineOliveiraa** [comment]
sim, @Tayling-Ng . se seu computador não tiver web cam, você deve utilizar o celular como câmera externa. No email que recebemos para agendar a prova e no site da CWI tem explicando bem direitinho!

**Comment by Askalor** [comment]
> sim, @Tayling-Ng . se seu computador não tiver web cam, você deve utilizar o celular como câmera externa. No email que recebemos para agendar a prova e no site da CWI tem explicando bem direitinho!

Existe algum outro requisito? Eu ainda não recebi o email (até estávamos falando sobre isso na outra issue sobre a prova) e gostaria de saber se precisarei de algo a mais para já ir me preparando.

**Comment by JaquelineOliveiraa** [comment]
@Askalor, vou te adiantar o que diz o email. Tenta acessar esse link pra agendar: https://reset.cwi.com.br/candidato

![Captura de Tela (5)](https://user-images.githubusercontent.com/100886754/191635460-3f675249-1ff0-4933-8d35-0d16edbd0003.png)
![Captura de Tela (6)](https://user-images.githubusercontent.com/100886754/191635464-c30e35b7-4059-4e90-8d94-735a377d9ae3.png)

**Comment by mariliaamorim** [comment]
Gostaria de saber qual a quantidade de questões?

**Comment by guilhermewmuller** [comment]
Pessoal, bom dia!
Vocês poderão sim utilizar 2 monitores, papel, caneta e demais anotações que tenham feito. Também não há restrição quanto ao uso da internet para pesquisas durante a prova. No entanto, avaliem se essa é a melhor estratégia considerando o tempo finito que terão.

**Comment by mariliaamorim** [comment]
Alguém sabe qual é a porcentagem de acertos para passar na prova?

**Comment by daiscrosa** [comment]
Bom dia pessoal,
Depois que fizeram o agendamento do horário da prova, vocês receberam algum e-mail de confirmação?
Ou mensagem na tela?
Quando agendei apareceu somente um visto e volta pra agendar, fiquei em dúvida se deu certo.

**Comment by Lindamayra** [comment]
> Bom dia pessoal, Depois que fizeram o agendamento do horário da prova, vocês receberam algum e-mail de confirmação? Ou mensagem na tela? Quando agendei apareceu somente um visto e volta pra agendar, fiquei em dúvida se deu certo.

A confirmação do agendamento será enviada por e-mail ainda hoje, juntamente com o link de acesso a prova. Apenas o último registro de agendamento será considerado.

**Comment by daiscrosa** [comment]
Obrigada!
________________________________
De: Lindamayra ***@***.***>
Enviado: quinta-feira, 22 de setembro de 2022 12:11
Para: cwi-reset/edicao-04-level-1 ***@***.***>
Cc: Daiane Costa da Rosa ***@***.***>; Comment ***@***.***>
Assunto: Re: [cwi-reset/edicao-04-level-1] Prova.. (Issue #360)


Bom dia pessoal, Depois que fizeram o agendamento do horário da prova, vocês receberam algum e-mail de confirmação? Ou mensagem na tela? Quando agendei apareceu somente um visto e volta pra agendar, fiquei em dúvida se deu certo.

A confirmação do agendamento será enviada por e-mail ainda hoje, juntamente com o link de acesso a prova. Apenas o último registro de agendamento será considerado.

**Comment by RebecaPR** [comment]
boa tarde gente! 
alguém já recebeu o e-mail de confirmação do horário da prova com o link?

**Comment by AdrianaFuzina** [comment]
Até agora, eu não recebi, estou preocupada.

**Comment by RebecaPR** [comment]
eu tb não...eles iam confirmar hoje a tarde

**Comment by tikaoo** [comment]
TB ainda não recebi...

**Comment by daiscrosa** [comment]
Oi

Nada aqui...
________________________________
De: Rebeca Ponciano ***@***.***>
Enviado: quinta-feira, 22 de setembro de 2022 18:03
Para: cwi-reset/edicao-04-level-1 ***@***.***>
Cc: Daiane Costa da Rosa ***@***.***>; Comment ***@***.***>
Assunto: Re: [cwi-reset/edicao-04-level-1] Prova.. (Issue #360)


boa tarde gente!
alguém já recebeu o e-mail de confirmação do horário da prova com o link?

**Comment by igoroliveirasilva** [comment]
Ainda não.

**Comment by Cristiano-Fleck** [comment]
Bom dia!

Eu agendei para o primeira hora da manhã e não recebi o e-mail de confirmação com o acesso a prova, já revirei a caixa de e-mail e nada.
